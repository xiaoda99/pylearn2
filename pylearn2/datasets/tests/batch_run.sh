#python test_cloudflowhd.py 2 2 3 31 8 2
#python test_cloudflowhd.py 1 2 6 31 16 4
#python test_cloudflowhd.py 1 2 6 31 20 5
python test_cloudflowhd.py 1 1 6 61 20 5
#python test_cloudflowhd.py 1 1 6 101 20 5
