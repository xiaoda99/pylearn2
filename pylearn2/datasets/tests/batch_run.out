Loading data for radar_img_matrix_AZ9280_201407_uint8.pkl.gz from ramdisk...
Done. 10.6714508533 seconds
Downsampling...
Done. 14.6839129925 seconds
Generating random examples ...
Done. 66.9757471085 seconds
Loading data for radar_img_matrix_AZ9280_201408_uint8.pkl.gz from ramdisk...
Done. 26.540941 seconds
Downsampling...
Done. 18.3792710304 seconds
Generating random examples ...
Done. 66.2224128246 seconds
Loading data for radar_img_matrix_AZ9280_201409_uint8.pkl.gz from ramdisk...
Done. 25.1264131069 seconds
Downsampling...
Done. 18.1718099117 seconds
Generating random examples ...
Done. 67.3835189342 seconds
Loading data for radar_img_matrix_AZ9010_201406_uint8.pkl.gz from ramdisk...
Done. 25.137283802 seconds
Downsampling...
Done. 17.831635952 seconds
Generating random examples ...
Done. 68.4234650135 seconds
Loading data for radar_img_matrix_AZ9010_201407_uint8.pkl.gz from ramdisk...
Done. 25.088023901 seconds
Downsampling...
Done. 16.2941019535 seconds
Generating random examples ...
Done. 68.3678050041 seconds
Loading data for radar_img_matrix_AZ9010_201408_uint8.pkl.gz from ramdisk...
Done. 24.6426010132 seconds
Downsampling...
Done. 17.3488130569 seconds
Generating random examples ...
Done. 68.8445529938 seconds
Loading data for radar_img_matrix_AZ9010_201409_uint8.pkl.gz from ramdisk...
Done. 24.9874899387 seconds
Downsampling...
Done. 18.6545460224 seconds
Generating random examples ...
Done. 68.4457628727 seconds
Loading data for radar_img_matrix_AZ9200_201406_uint8.pkl.gz from ramdisk...
Done. 25.0453548431 seconds
Downsampling...
Done. 18.7659139633 seconds
Generating random examples ...
Done. 68.3938219547 seconds
Loading data for radar_img_matrix_AZ9200_201407_uint8.pkl.gz from ramdisk...
Done. 25.2096569538 seconds
Downsampling...
Done. 18.5964980125 seconds
Generating random examples ...
Done. 68.5521709919 seconds
Loading data for radar_img_matrix_AZ9200_201408_uint8.pkl.gz from ramdisk...
Done. 25.0529448986 seconds
Downsampling...
Done. 18.4318749905 seconds
Generating random examples ...
Done. 68.2672178745 seconds
Loading data for radar_img_matrix_AZ9200_201409_uint8.pkl.gz from ramdisk...
Done. 25.115434885 seconds
Downsampling...
Done. 17.5484127998 seconds
Generating random examples ...
Done. 68.5635111332 seconds
exact flow
2:	0.832	0.797	0.814
4:	0.716	0.684	0.700
6:	0.638	0.609	0.623
8:	0.576	0.544	0.560
approximate mean flow
2:	0.837	0.794	0.815
4:	0.724	0.684	0.703
6:	0.647	0.606	0.626
8:	0.583	0.538	0.560
Loading data for radar_img_matrix_AZ9280_201407_uint8.pkl.gz from ramdisk...
Done. 21.8576869965 seconds
Downsampling...
Done. 16.6533391476 seconds
Generating random examples ...
Done. 176.093665123 seconds
Loading data for radar_img_matrix_AZ9280_201408_uint8.pkl.gz from ramdisk...
Done. 24.9271550179 seconds
Downsampling...
Done. 22.8170440197 seconds
Generating random examples ...
Done. 174.004966021 seconds
Loading data for radar_img_matrix_AZ9280_201409_uint8.pkl.gz from ramdisk...
Done. 25.0965278149 seconds
Downsampling...
Done. 21.8457260132 seconds
Generating random examples ...
Done. 175.341115952 seconds
Loading data for radar_img_matrix_AZ9010_201406_uint8.pkl.gz from ramdisk...
Done. 24.9623250961 seconds
Downsampling...
Done. 20.4449908733 seconds
Generating random examples ...
Done. 179.018465996 seconds
Loading data for radar_img_matrix_AZ9010_201407_uint8.pkl.gz from ramdisk...
Done. 24.8752248287 seconds
Downsampling...
Done. 19.6082260609 seconds
Generating random examples ...
Done. 178.458564043 seconds
Loading data for radar_img_matrix_AZ9010_201408_uint8.pkl.gz from ramdisk...
Done. 24.8858938217 seconds
Downsampling...
Done. 19.0811707973 seconds
Generating random examples ...
Done. 178.652586937 seconds
Loading data for radar_img_matrix_AZ9010_201409_uint8.pkl.gz from ramdisk...
Done. 25.0229420662 seconds
Downsampling...
Done. 21.4842581749 seconds
Generating random examples ...
Done. 179.078821898 seconds
Loading data for radar_img_matrix_AZ9200_201406_uint8.pkl.gz from ramdisk...
Done. 24.8811049461 seconds
Downsampling...
Done. 19.4430239201 seconds
Generating random examples ...
Done. 178.932977915 seconds
Loading data for radar_img_matrix_AZ9200_201407_uint8.pkl.gz from ramdisk...
Done. 24.8969380856 seconds
Downsampling...
Done. 20.4155440331 seconds
Generating random examples ...
Done. 179.582129955 seconds
Loading data for radar_img_matrix_AZ9200_201408_uint8.pkl.gz from ramdisk...
Done. 24.9973521233 seconds
Downsampling...
Done. 18.7346971035 seconds
Generating random examples ...
Done. 178.380000114 seconds
Loading data for radar_img_matrix_AZ9200_201409_uint8.pkl.gz from ramdisk...
Done. 25.3035511971 seconds
Downsampling...
Done. 20.2554731369 seconds
Generating random examples ...
Done. 179.677066088 seconds
exact flow
4:	0.848	0.788	0.817
8:	0.714	0.683	0.698
12:	0.621	0.609	0.615
16:	0.549	0.552	0.551
approximate mean flow
4:	0.868	0.778	0.821
8:	0.744	0.679	0.710
12:	0.657	0.603	0.629
16:	0.588	0.544	0.565
Loading data for radar_img_matrix_AZ9280_201407_uint8.pkl.gz from ramdisk...
Done. 26.4237270355 seconds
Downsampling...
Done. 16.8013720512 seconds
Generating random examples ...
Done. 206.234860897 seconds
Loading data for radar_img_matrix_AZ9280_201408_uint8.pkl.gz from ramdisk...
Done. 24.9470930099 seconds
Downsampling...
Done. 20.806524992 seconds
Generating random examples ...
Done. 204.086840868 seconds
Loading data for radar_img_matrix_AZ9280_201409_uint8.pkl.gz from ramdisk...
Done. 25.1431720257 seconds
Downsampling...
Done. 19.8998439312 seconds
Generating random examples ...
Done. 206.330913782 seconds
Loading data for radar_img_matrix_AZ9010_201406_uint8.pkl.gz from ramdisk...
Done. 25.0342240334 seconds
Downsampling...
Done. 19.868060112 seconds
Generating random examples ...
Done. 211.35160017 seconds
Loading data for radar_img_matrix_AZ9010_201407_uint8.pkl.gz from ramdisk...
Done. 24.9686532021 seconds
Downsampling...
Done. 18.8606829643 seconds
Generating random examples ...
Done. 211.135136127 seconds
Loading data for radar_img_matrix_AZ9010_201408_uint8.pkl.gz from ramdisk...
Done. 24.6397299767 seconds
Downsampling...
Done. 19.1689360142 seconds
Generating random examples ...
Done. 210.472985029 seconds
Loading data for radar_img_matrix_AZ9010_201409_uint8.pkl.gz from ramdisk...
Done. 25.0597720146 seconds
Downsampling...
Done. 18.3685131073 seconds
Generating random examples ...
Done. 209.922497034 seconds
Loading data for radar_img_matrix_AZ9200_201406_uint8.pkl.gz from ramdisk...
Done. 25.290776968 seconds
Downsampling...
Done. 19.4636039734 seconds
Generating random examples ...
Done. 209.329715967 seconds
Loading data for radar_img_matrix_AZ9200_201407_uint8.pkl.gz from ramdisk...
Done. 24.8768560886 seconds
Downsampling...
Done. 18.8367381096 seconds
Generating random examples ...
Done. 210.125967026 seconds
Loading data for radar_img_matrix_AZ9200_201408_uint8.pkl.gz from ramdisk...
Done. 24.9648990631 seconds
Downsampling...
Done. 19.5879199505 seconds
Generating random examples ...
Done. 208.264505863 seconds
Loading data for radar_img_matrix_AZ9200_201409_uint8.pkl.gz from ramdisk...
Done. 25.052216053 seconds
Downsampling...
Done. 19.8123729229 seconds
Generating random examples ...
Done. 210.634747982 seconds
exact flow
5:	0.833	0.774	0.802
10:	0.683	0.658	0.670
15:	0.582	0.577	0.580
20:	0.506	0.516	0.511
approximate mean flow
5:	0.859	0.762	0.808
10:	0.719	0.652	0.683
15:	0.622	0.570	0.595
20:	0.546	0.504	0.524
Loading data for radar_img_matrix_AZ9280_201407_uint8.pkl.gz from ramdisk...
Done. 26.2723360062 seconds
Generating random examples ...
Done. 145.326329947 seconds
Loading data for radar_img_matrix_AZ9280_201408_uint8.pkl.gz from ramdisk...
Done. 27.1985201836 seconds
Generating random examples ...
Done. 137.577983856 seconds
Loading data for radar_img_matrix_AZ9280_201409_uint8.pkl.gz from ramdisk...
Done. 26.2192869186 seconds
Generating random examples ...
Done. 138.007923841 seconds
Loading data for radar_img_matrix_AZ9010_201406_uint8.pkl.gz from ramdisk...
Done. 25.4269728661 seconds
Generating random examples ...
Done. 141.589747906 seconds
Loading data for radar_img_matrix_AZ9010_201407_uint8.pkl.gz from ramdisk...
Done. 25.4293699265 seconds
Generating random examples ...
Done. 141.930850029 seconds
Loading data for radar_img_matrix_AZ9010_201408_uint8.pkl.gz from ramdisk...
Done. 25.3929297924 seconds
Generating random examples ...
Done. 140.807493925 seconds
Loading data for radar_img_matrix_AZ9010_201409_uint8.pkl.gz from ramdisk...
Done. 25.8196189404 seconds
Generating random examples ...
Done. 141.732067108 seconds
Loading data for radar_img_matrix_AZ9200_201406_uint8.pkl.gz from ramdisk...
Done. 25.468173027 seconds
Generating random examples ...
Done. 138.993178129 seconds
Loading data for radar_img_matrix_AZ9200_201407_uint8.pkl.gz from ramdisk...
Done. 25.3956370354 seconds
Generating random examples ...
Done. 140.149370909 seconds
Loading data for radar_img_matrix_AZ9200_201408_uint8.pkl.gz from ramdisk...
Done. 25.4374921322 seconds
Generating random examples ...
Done. 139.691162825 seconds
Loading data for radar_img_matrix_AZ9200_201409_uint8.pkl.gz from ramdisk...
Done. 26.106978178 seconds
Generating random examples ...
Done. 142.008639097 seconds

python test_cloudflowhd.py 1 1 6 61 20 5
exact flow
5:	0.829	0.768	0.797
10:	0.684	0.655	0.669
15:	0.589	0.578	0.583
20:	0.520	0.523	0.521
approximate mean flow
5:	0.857	0.758	0.805
10:	0.724	0.647	0.684
15:	0.632	0.570	0.599
20:	0.563	0.513	0.537

python test_cloudflowhd.py 1 1 6 101 20 5
exact flow
5:	0.822	0.760	0.790
10:	0.674	0.643	0.658
15:	0.582	0.568	0.575
20:	0.515	0.513	0.514
approximate mean flow
5:	0.866	0.741	0.798
10:	0.736	0.631	0.680
15:	0.650	0.556	0.600
20:	0.583	0.500	0.538

exact flow
5:	0.828	0.767	0.796
10:	0.684	0.653	0.668
15:	0.589	0.579	0.584
20:	0.519	0.520	0.520
exact mean flow
5:	0.857	0.759	0.805
10:	0.730	0.649	0.687
15:	0.643	0.576	0.608
20:	0.577	0.517	0.546
approximate mean flow
5:	0.857	0.758	0.804
10:	0.724	0.646	0.683
15:	0.632	0.573	0.601
20:	0.563	0.511	0.536

